homeassistant/components/imeon_inverter/sensor.py

```diff
@@ -0,0 +1,79 @@
+"""Imeon inverter sensor support."""
+
+import logging
+
+from homeassistant.components.sensor import SensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import InverterCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Create each sensor for a given config entry."""
+
+    # Get Inverter from UUID
+    IC: InverterCoordinator = entry.runtime_data
+
+    async_add_entities(
+        ([InverterSensor(IC, "meter_power", entry, "Meter Power", "W")]), True
+    )
+
+
+class InverterSensor(CoordinatorEntity, SensorEntity):
+    """A sensor that returns numerical values with units."""
+
+    def __init__(self, coordinator, data_key, entry, friendly_name, unit) -> None:
+        """Pass coordinator to CoordinatorEntity."""
+        super().__init__(coordinator)
+        self.data_key = data_key
+        self._entry_id = entry.entry_id
+        self._namespace = DOMAIN + "." + data_key
+        self._device = entry.title
+
+        self._attr_name = friendly_name
```

Comment:
put name in strings.json

Reply:
how can i access to a string stored in strings.json ?

Reply:
you set `_attr_translation_key` the rest happens automatically